src/types/grunnlagsdataOgVilkårsvurderinger/grunnlagsdataOgVilkårsvurderinger.ts

```diff
@@ -10,6 +11,7 @@ import { UføreVilkår } from './uføre/Uførevilkår';
 import { Utenlandsopphold } from './utenlandsopphold/Utenlandsopphold';
 
 export interface GrunnlagsdataOgVilkårsvurderinger {
+    alder: Nullable<Aldersvilkår>;
     uføre: Nullable<UføreVilkår>;
```

Comment:
vi får alltid tilbake begge to nå?

Reply:
Nope, avhenger av type. Så vi må vurdere å kjøre generisk (eller den ikke generiske vi gikk for på søknadsinnhold) her og